commit 4070aed9e7041e75e0bd5d707fa34ec2ae76d021
Author: Sean Young <sean@mess.org>
Date:   Fri Dec 2 15:16:08 2016 -0200

    lirc_dev: LIRC_{G,S}ET_REC_MODE do not work
    
    commit bd291208d7f5d6b2d6a033fee449a429230b06df upstream.
    
    Since "273b902 [media] lirc_dev: use LIRC_CAN_REC() define" these
    ioctls no longer work.
    
    Signed-off-by: Sean Young <sean@mess.org>
    Reviewed-by: Andi Shyti <andi.shyti@samsung.com>
    Signed-off-by: Mauro Carvalho Chehab <mchehab@s-opensource.com>
    Signed-off-by: Greg Kroah-Hartman <gregkh@linuxfoundation.org>

Index: linuxtest/dummy/rpi_1225_4070aed9e7041e75e0bd5d707fa34ec2ae76d021.txt
===================================================================
--- /dev/null
+++ linuxtest/dummy/rpi_1225_4070aed9e7041e75e0bd5d707fa34ec2ae76d021.txt
@@ -0,0 +1 @@
+dummy file to ensure patch has content.
